addons/mail/static/tests/owl/component/chatter_tests.js

```diff
@@ -117,6 +118,47 @@ QUnit.test('base rendering when chatter has no attachments', async function (ass
     );
 });
 
+QUnit.test('base rendering when chatter has no record', async function (assert) {
+    assert.expect(7);
+    await this.start({});
+    const chatterLocalId = this.env.store.dispatch('createChatter', { initialThreadModel: 'res.partner' });
+    await this.createChatterComponent({ chatterLocalId });
+    assert.strictEqual(
+        document.querySelectorAll(`.o_Chatter`).length,
+        1,
+        "should have a chatter"
+    );
+    assert.strictEqual(
+        document.querySelectorAll(`.o_ChatterTopbar`).length,
+        1,
+        "should have a chatter topbar"
+    );
+    assert.strictEqual(
+        document.querySelectorAll(`.o_Chatter_attachmentBox`).length,
+        0,
+        "should not have an attachment box in the chatter"
+    );
+    assert.strictEqual(
+        document.querySelectorAll(`.o_Chatter_thread`).length,
+        1,
+        "should have a thread in the chatter"
+    );
+    assert.ok(
+        document.querySelector(`.o_Chatter_thread`).classList.contains('o-is-temporary'),
+        "thread should have a temporary thread local id"
+    );
```

Comment:
as said before, I prefer looking for thread in store it is temporary. Better trust the store than the rendering of component